crates/transaction-fuzzer/src/account_universe/universe.rs

```diff
@@ -38,7 +40,7 @@ pub struct AccountUniverse {
 /// Allows pairs of accounts to be uniformly randomly selected from an account universe.
```

Comment:
We may want to update the comments and struct / function names since we now allow any PICK_SIZE.